service/service.py

```diff
@@ -74,6 +87,25 @@
     "status", type=inputs.boolean, required=False, help="List Recommendations by status"
 )
 
+
+######################################################################
+# Authorization Decorator
```

Comment:
I think we agreed on not having an API key right?

Reply:
Ah yes, I forgot to clean this part up. Thanks for reminding, I will delete this now.

Reply:
> I think we agreed on not having an API key right?
Ah yes, I forgot to clean this part up. Thanks for reminding, I will delete this now.